docs/index.md

```diff
@@ -2363,7 +2407,7 @@ There are typically many assertions, so this is a paged view.
 
 After that begin the list of assertions (theorems and axioms).
 Each assertion shows its count, the type of assertion
-(theorem or axiom), and the assertion id in bold.
+(theorem or axiom), and the assertion label (id) in bold.
```

Comment:
`the assertion label (id)`

`id` is never used on the UI with same meaning as label. Even more, `id` is never used on UI at all. So, probably, mentioning of ids could be removed from the guide.

`id` is appearing in the mm-lamp's code, but it is something different from labels.